apps/connector/Jenkinsfile

```diff
@@ -28,40 +28,60 @@ pipeline {
 
   stages {
     stage('Install') {
-      steps { script {
-        nix.shell('yarn install --frozen-lockfile', pure: false)
-      } }
+      steps {
+          dir("${env.WORKSPACE}/apps/connector") {
+            script {
+              withEnv(["WORKSPACE=${env.WORKSPACE}/apps/connector"]) { // Override WORKSPACE
```

Comment:
That's way too overcomplicated. Wouldn't it be better to just add support for an `entrypoint` argument for `nix.shell()` and other commands in `status-jenkins-lib`?

Reply:
I agree with that, will fix it right away